### DISCUSSION TITLE
Calculating Janet Repositories on GitHub

### DISCUSSION THREAD

**Discussion by pyrmont** [janet-lang/janet#830] [Show and tell]
I've released a small project, [Is Janet Popular?](https://github.com/pyrmont/is-janet-popular) It uses GitHub's (honestly quite terrible) code search API to make a rough calculation of the number of repositories on GitHub that use Janet.

It undercounts (did I mention GitHub's code search API is terrible?) but if you've been itching to know how Janet is doing and you're despondent [Is Janet There Yet?](https://isjanetthereyet.herokuapp.com) hasn't run since June, this might be the tool for you. 

If you just want to see the stats, I'm planning on running it weekly and putting the output [here](https://inqk.net/files/is_janet_popular.json).

**Comment by sogaiu** [comment]
On a side note, there were some things there I hadn't seen before.

Thanks!

**Reply by pepe** [reply]
Some examples please.

**Reply by sogaiu** [reply]
https://github.com/tmward/janet-misc
https://github.com/tmward/jask
https://github.com/Luexa/jvs
https://github.com/MorganPeterson/jermbox
https://github.com/MorganPeterson/tabletree
